GameModes/OnlineGameModeHelpers.cs

```diff
@@ -12,7 +12,7 @@ public static class OnlineGameModeHelpers
             //PlacedObject.Type.PuffBall,
             //PlacedObject.Type.TempleGuard,
             PlacedObject.Type.LightFixture,
-            //PlacedObject.Type.DangleFruit,
+            PlacedObject.Type.DangleFruit,
```

Comment:
Fruits aren't cosmetic. Lets figure out a way to fit this into our gamemode API, if we need more methods/filters lets make them!